bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSessionSharedThreadPools.java

```diff
@@ -81,6 +83,10 @@ public ExecutorService getBatchThreadPool() {
     return batchThreadPool;
   }
 
+  public ListeningExecutorService getListeningExecutorService() {
```

Comment:
Not sure this belongs to BigtableSessionSharedThreadPools, it doesn't really return any new object, just another flavor of the same object as getBatchThreadPool().